components/graphs/Graph.tsx

```diff
@@ -56,21 +72,26 @@ const Graph: FC<{ siteUUID: string }> = ({ siteUUID }) => {
     updateEnabled: timeEnabled,
   });
 
+  const [timeRange, setTimeRange] = useState('24');
```

Comment:
Replace this string with a number so you don't have to do `parseInt` down below. There's almost no difference between using a number or a string here, doing it this way will just reduce the number of `parseInt` calls.
```suggestion
  const [timeRange, setTimeRange] = useState(24);
```